test/commands/info/releasenotes/display.test.ts

```diff
@@ -60,26 +66,20 @@ describe('info:releasenotes:display', () => {
     oclifConfigStub.root = '/root/path';
 
     suppressEnvVarBackup = process.env.PLUGIN_INFO_HIDE_RELEASE_NOTES;
+    suppressFooterBackup = process.env.PLUGIN_INFO_HIDE_FOOTER;
 
     getInfoConfigStub = stubMethod(sandbox, getInfoConfig, 'getInfoConfig').returns(mockInfoConfig);
-    getReleaseNotesStub = stubMethod(sandbox, getReleaseNotes, 'getReleaseNotes').returns('release-notes');
+    getReleaseNotesStub = stubMethod(sandbox, getReleaseNotes, 'getReleaseNotes').returns('## Release notes for 3.3.3');
     getDistTagVersionStub = stubMethod(sandbox, getDistTagVersion, 'getDistTagVersion').returns('1.2.3');
+    parseReleaseNotesSpy = spyMethod(sandbox, parseReleaseNotes, 'parseReleaseNotes');
+    markedParserSpy = spyMethod(sandbox, marked, 'parser');
   });
 
   afterEach(() => {
-    getInfoConfigStub.restore();
-    getReleaseNotesStub.restore();
-    getDistTagVersionStub.restore();
     sandbox.restore();
 
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    process.env.PLUGIN_INFO_HIDE_RELEASE_NOTES = suppressEnvVarBackup;
-  });
-
-  it('logs release notes', async () => {
-    await runDisplayCmd([]);
-
-    expect(uxLogStub.args[0][0]).to.equal('release-notes');
+    process.env.PLUGIN_INFO_HIDE_RELEASE_NOTES = suppressEnvVarBackup; // eslint-disable-line @typescript-eslint/no-unsafe-assignment
```

Comment:
is there a way to avoid actually manipulating the envs in tests?  It can cause problems when things run in parallel and can be pretty hard to debug

Mocking the `env.get()` stuff maybe?

Reply:
Ah, good call. Didn't think about running parallel test. Will stub `env`.